app/Http/Controllers/v1/CategoryController.php

```diff
@@ -30,7 +29,7 @@ public function index()
 
         $query->orderBy($sortBy, $sortOrder);
 
-        $categories = $query->paginate($perPage);
+        $categories = $query->get();
```

Comment:
What is the reason we are getting rid of pagination here?

Reply:
Nokie suggested that we will have never enough categories to paginate.

Reply:
makes sense :+1: